resources/local/config.toml

```diff
@@ -21,6 +21,8 @@ control_logic_default_delay = '1sec'
 # Flag which forces the node to resync all of the blocks.
 force_resync = false
 
+# Maximum time a node will wait for an upgrade to commit
```

Comment:
```suggestion
# Maximum time a node will wait for an upgrade to commit.
```

Reply:
Fixed in https://github.com/casper-network/casper-node/pull/3971/commits/2b4c9ae04b1ad1c97b873fabb58367cc72e32144